client/src/legacy/HistoryViewer/HistoryViewerEntwine.js

```diff
@@ -20,6 +20,7 @@ jQuery.entwine('ss', ($) => {
       ReactDOM.render(
```

Comment:
The lack of an `onunmatch` event here is causing a critical bug, where each match of the selector is loading another HistoryViewer component into the stack, meaning the number of renders (and network requests to the form schema) are endlessly building. 

```js
    onunmatch() {
      ReactDOM.unmountComponentAtNode(this[0]);
    }
```

This is the current output when viewing a detail after having previously viewed two other pages:

![screenshot 2018-04-17 11 42 20](https://user-images.githubusercontent.com/654636/38840564-6a32fd78-4234-11e8-89bb-00c4a0d89985.png)

Reply:
True! This was logged as #7 but I'll fix it up in this PR too